cumulusci/tasks/apex/testrunner.py

```diff
@@ -179,11 +181,16 @@ def _init_options(self, kwargs):
         self.options["test_name_match"] = self.options.get(
             "test_name_match", self.project_config.project__test__name_match
         )
-
         self.options["test_name_exclude"] = self.options.get(
             "test_name_exclude", self.project_config.project__test__name_exclude
         )
 
+        self.options["test_suite_names"] = self.options.get(
```

Comment:
Let's use the `process_list_arg()` utility function here (you can find examples in the codebase). That will handle a situation where the user provided a comma-separated list on the command line, rather than a structured list in the YAML configuration.

Reply:
Hey David, currently, input is being fetched from the command line. Input is not read from YAML configuration as of yet. Comma-separated list provided by the user is successfully processed by the utility function _get_comma_separated_string_of_items which formats the list of strings in to a single string which is subsequently inserted into queries. 

Please let us know if we are missing anything here.